test/views/sections/items/ItemPackagingSealChoiceViewSpec.scala

```diff
@@ -44,15 +44,16 @@ class ItemPackagingSealChoiceViewSpec extends SpecBase with ViewBehaviours {
         val view = app.injector.instanceOf[ItemPackagingSealChoiceView]
         val form = app.injector.instanceOf[ItemPackagingSealChoiceFormProvider].apply()
 
-        implicit val doc: Document = Jsoup.parse(view(form, testOnwardRoute, "Aerosol").toString())
+        implicit val doc: Document = Jsoup.parse(view(form, testOnwardRoute, "Aerosol", 0, 0, "1", false).toString())
 
         behave like pageWithExpectedElementsAndMessages(Seq(
           Selectors.title -> messagesForLanguage.title,
           Selectors.subHeadingCaptionSelector -> messagesForLanguage.itemSection,
           Selectors.h1 -> messagesForLanguage.heading,
           Selectors.p(1) -> messagesForLanguage.p1,
-          Selectors.p(2) -> messagesForLanguage.p2("Aerosol"),
-          Selectors.strong(1) -> "Aerosol",
+          Selectors.p(2) -> messagesForLanguage.p2("1","1","Aerosol","1"),
+          Selectors.strong(1) -> "Aerosol:",
+          Selectors.strong(2) -> "1",
```

Comment:
If you make the change as suggested in the view, only one strong selector would be needed